guides/scripts/generate-hammer-reference.sh

```diff
@@ -72,24 +71,23 @@ done
 SRC=$1
 
 # Define internal variables
-if [ -n "$ASMB_DIR" ]; then
-    mod_path=$ASMB_DIR/$MOD_DIR # module destination folder
-    asmb_file=$ASMB_DIR/$ASMB_FILENAME
-else
-    mod_path=$MOD_DIR
-    asmb_file=$ASMB_FILENAME
-fi
+mod_path=${MOD_DIR%/}
+asmb_file=$ASMB_FILENAME
 details_tmp="$mod_path/tmp_hammer-option-details"
 echo -e >$details_tmp # reset
 
 details_file="$mod_path/ref_hammer-option-details.adoc"
-details_header='[id="hammer-option-details"]\n= Option details\n\nHammer options accept the following option types and values:'
+details_header="$MOD_TYPE\n\n[id="hammer-option-details"]\n= Option details\n\n$ROLE_ABSTRACT\nHammer options accept the following option types and values:"
 
-asmb_header=':_mod-docs-content-type: ASSEMBLY\n\n// DO NOT EDIT MANUALLY\n// Use the generate-hammer-reference.sh script to update\n\ninclude::modules/con_hammer-reference.adoc[]\n\n'
-asmb_footer='include::modules/ref_hammer-option-details.adoc[leveloffset=+1]\n'
+asmb_header='include::common/attributes.adoc[]\ninclude::common/header.adoc[]\n:context: hammer-reference\n:hammer-reference:\n\n= {HammerRefDocTitle}\n\nifdef::satellite[]\ninclude::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]\nendif::[]\n\n// DO NOT EDIT MANUALLY!\n// Use the generate-hammer-reference.sh script to update\n'
+asmb_footer='include::common/modules/ref_hammer-option-details.adoc[leveloffset=+1]\n\nifndef::orcharhino,satellite[]\ninclude::common/ribbons.adoc[]\nendif::[]\n'
 
-# Create folders
-{ [ -n "$NOOP" ] || mkdir -p $mod_path ; } && echo -e "Created folders: " $mod_path
+# Create folders if they don't exist
+if [ -d "$mod_path" ]; then
+    echo "I: Module path exists: $mod_path"
+else
+    { [ -n "$NOOP" ] || mkdir -p $mod_path ; } && echo -e "I: Created path: $mod_path"
```

Comment:
Should the script fail if the folder doesn't exist?

Reply:
Probably, yes. It's an indication that the user is running the script in the wrong place, isn't it?